Makefile.in

```diff
@@ -388,9 +387,16 @@ endif
 	@echo "Missing xsltproc.  "$@" not (re)built."
 endif
 
-$(objroot)doc/%.3 : $(objroot)doc/%.xml $(srcroot)doc/stylesheet.xsl $(objroot)doc/manpages.xsl
+$(objroot)doc/%$(install_suffix).3 : $(objroot)doc/%.xml $(srcroot)doc/stylesheet.xsl $(objroot)doc/manpages.xsl
 ifneq ($(XSLROOT),)
 	$(XSLTPROC) -o $@ $(objroot)doc/manpages.xsl $<
+# The -o option (output filename) of xsltproc may not work (it uses the
+# <refname> in the .xml file).  Manually add the suffix if so.
```

Comment:
This gave me a pause. Found that the generated `.3` always uses the name here:
https://github.com/jemalloc/jemalloc/blob/dev/doc/jemalloc.xml.in#L27, and the `-o filename` has no effect at all.

Therefore the workaround below.

Reply:
Thanks for the explanation